TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -43,10 +44,26 @@ object TeleOpMovement : Component {
         var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
         // if the right bumper is pressed move slower
-        if (gamepad.right_bumper) {
+        if (gamepad.y) {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // Inputs for the gripper on the box
+        if (gamepad.left_bumper) {
+            Box.gripIn()
+        }
+        if (gamepad.right_bumper) {
+            Box.gripOut()
+        }
+
+        // Inputs for the movement of the box
+        if (gamepad.a) {
+            Box.dropBox()
+        }
+        if (gamepad.b) {
+            Box.pickUpBox()
+        }
```

Comment:
Same likewise here. Could this be an `else if` statement?

Reply:
corrected